src/routes/Home.js

```diff
@@ -302,8 +302,8 @@ export default props => (
     </Hero>
     <ShortNameAuctions>
       <h3>
-        The short name reservations are in progress at&nbsp;
-        <a href="https://reserve.ens.domains">reserve.ens.domains</a>
+        The short name auctions are in progress at&nbsp;
+        <a href="https://opensea.io/ens-landing">opensea.io/ens-landing</a>
```

Comment:
Same as above. better to open new tab

Reply:
Don't think it's necessary to open a new tab since the manager does not support the auction

Reply:
We open new tab when people click external link so it is consistent to provide the same behavior. It's irrelevant whether the auction starts or not.

Reply:
It is relevant. Because on mobile it will open a new tab and that is not a great experience.